Attaching_Images_To_Data.md

```diff
@@ -0,0 +1,26 @@
+# A tutorial on uploading Images
+
+## Where do you need to upload images? 
+Whenever you're sharing a resource, or a community event, there is an option to share an image link in the data you're uploading, which is basically a JSON object. For example, if you're adding a new event, your content may look like: 
+
+```
+{
+        "title": "Outreachy 2023",
+        "image": "https://user-images.githubusercontent.com/78233115/194045366-942e7379-92ba-4625-9048-939e4bba434c.png",
+        "description": "The Outreachy program is a really great program for students that aims to support diversity in open source and open science. Outreachy provides internships to people subject to systemic bias and impacted by underrepresentation in the technical industry where they are living.",
+        "url": "https://www.outreachy.org/",
+        "end_date": "Application will open in early February-2023 and will end in late February-2023"
```

Comment:
Just change the end_date

'year-month-day'

"2023-02-25"

Reply:
Done!